services/app-web/src/components/Modal/V2/UnlockSubmitModalV2.tsx

```diff
@@ -31,9 +34,15 @@ const RATE_UNLOCK_SUBMIT_TYPES = [
     'RESUBMIT_RATE',
     'UNLOCK_RATE',
 ] as const
+const CONTRACT_UNLOCK_SUBMIT_TYPES = [
+    'SUBMIT_CONTRACT',
+    'RESUBMIT_CONTRACT',
+    'UNLOCK_CONTRACT',
+] as const
 type PackageModalType = (typeof PACKAGE_UNLOCK_SUBMIT_TYPES)[number]
 type RateModalType = (typeof RATE_UNLOCK_SUBMIT_TYPES)[number]
-type SharedModalType = PackageModalType & RateModalType
+type ContractModalType = (typeof CONTRACT_UNLOCK_SUBMIT_TYPES)[number]
+type SharedModalType = PackageModalType & ContractModalType & RateModalType
```

Comment:
We can fully delete the package modal types - this new v2 will be only contract or rate. I think I just didn't get that yet in the refactor.